source/common/clusterWorker.js

```diff
@@ -261,26 +312,29 @@ module.exports = function (spec) {
                 'keepAlive',
                 [id],
                 function (result) {
-                    loss_count = 0;
+                    //keycoding 20230811: fix when a newer leader was vote,the agent must pickUp itself
                     if (result === 'whoareyou') {
+                        loss_count += 1;
+                        if (loss_count % 300 === 0 || loss_count === 3) {
```

Comment:
Why % 300 === 0 here?
Is line 318 duplicated with line 336?

Reply:
when the agent is found to be lost，the first line of logs is printed, and every 5 minutes, one line of logs is still lost
i had add more comment in the new commit